examples/gno.land/r/gov/dao/prop1_filetest.gno

```diff
@@ -1,11 +1,13 @@
+// PKGPATH: gno.land/r/foo/prop01
+//
 // Please note that this package is intended for demonstration purposes only.
 // You could execute this code (the init part) by running a `maketx run` command
 // or by uploading a similar package to a personal namespace.
 //
 // For the specific case of validators, a `r/gnoland/valopers` will be used to
 // organize the lifecycle of validators (register, etc), and this more complex
 // contract will be responsible to generate proposals.
-package main
+package prop01
```

Comment:
Is this filetest being executed when calling `gno test` ?

Reply:
```console
$> go run github.com/gnolang/gno/gnovm/cmd/gno test -v --update-golden-tests --print-runtime-metrics ./gov/dao
=== RUN   file/prop1_filetest.gno
--- PASS: file/prop1_filetest.gno (0.93s)
=== RUN   file/prop2_filetest.gno
--- PASS: file/prop2_filetest.gno (1.06s)
ok      ./gov/dao       1.99s
```